integration/simulation/network/obscuro_node_utils.go

```diff
@@ -117,8 +116,7 @@ func startStandaloneObscuroNodes(params *params.SimParams, gethClients []ethadap
 
 	// start each obscuro node
 	for _, m := range obscuroNodes {
-		t := m
-		go t.Start()
+		_ = m.Start()
```

Comment:
Worth checking the err response and panicking here? I think at the moment it would just silently swallow it if something went wrong at this point in the sim.

Reply:
I would think It would too, but it might be misleading as errors are not being bubbled up.
Added a todo and a ticket. WTDT ?

Reply:
What do you think would be misleading? At the very least I'd log it at ERROR level `"node might not have started properly, container errored - %w"`

I'm just worried that there might be unpredictable behaviour from a node in a bad state, so it might not fail the smoke tests but then have difficult to diagnose issues later for example.

Reply:
Nothing will ever error from `_ = m.Start()`.
`m.Start()` calls `HostContainer.Start()` which always returns nil. 
And inside `HostContainer.Start()`, the `Start()` calls of each of the components do not return error. They instead rely on panic'ing.

That's why I think adding an error check here is misleading as the errors are never bubbled up.
It totally should be changed, but maybe in a different PR?